lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/failed.yaml

```diff
@@ -57,42 +57,28 @@ states:
   FAILED_UPDATE_DETAILS:
     events:
       next:
-        targetState: CRI_TICF_BEFORE_COULD_NOT_CONFIRM_DETAILS
+        targetState: CRI_TICF_BEFORE_COULD_NOT_UPDATE_DETAILS
         auditEvents:
           - IPV_USER_DETAILS_UPDATE_END
         auditContext:
           successful: false
         checkIfDisabled:
           ticf:
-            targetState: COULD_NOT_CONFIRM_DETAILS_PAGE
+            targetState: COULD_NOT_UPDATE_DETAILS_PAGE
             auditEvents:
               - IPV_USER_DETAILS_UPDATE_END
             auditContext:
               successful: false
             checkJourneyContext:
               rfc:
-                targetState: COULD_NOT_CONFIRM_DETAILS_PAGE
-                auditEvents:
-                  - IPV_USER_DETAILS_UPDATE_END
-                auditContext:
-                  successful: false
-            checkFeatureFlag:
-              updateDetailsAccountDeletion:
-                targetState: COULD_NOT_UPDATE_DETAILS_PAGE
+                targetState: COULD_NOT_UPDATE_DETAILS_PAGE_RFC
                 auditEvents:
                   - IPV_USER_DETAILS_UPDATE_END
                 auditContext:
                   successful: false
         checkJourneyContext:
           rfc:
-            targetState: CRI_TICF_BEFORE_COULD_NOT_CONFIRM_DETAILS
```

Comment:
might be my unfamiliarity with how the `journeyContext` works (see [this](https://github.com/govuk-one-login/ipv-core-back/pull/2367/files#r1740647559) comment) but if `FAILED_UPDATE_DETAILS` only has two variants (reuse or rfc), would handling the `checkFeatureFlag` be better in the target state?

Reply:
You could - and that would remove the need for a separate `CRI_TICF_BEFORE_COULD_NOT_UPDATE_DETAILS_RFC`

On the other hand, this approach means that all the context checking is in one place (this entry state) :shrug: